bin/cleanup_workspace

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+Dir.chdir(File.dirname(__dir__))
+
+require_relative '../lib/auto_hck'
+
+module AutoHCK
+  class WorkspaceCleaner
+    def initialize(cleanup_time)
+      @cleanup_time = cleanup_time || 2_592_000 # Default to 30 days
+    end
+
+    def self.cleanup(cleanup_time)
+      new(cleanup_time).cleanup
+    end
```

Comment:
This method is too trivial

Reply:
I make it this way for better reading of this `WorkspaceCleaner.cleanup(cleanup_time)`

Reply:
I don't think it's significant enough. If this method is not present, you will need write: `WorkspaceCleaner.new.cleanup(cleanup_time)`. I don't think the presence of `.new` matters.